### DISCUSSION TITLE
CRT shaders? Nostalgia BIOS intros? Are those hardcore stuff?

### DISCUSSION THREAD

**Discussion by mffakhri** [Swordfish90/Lemuroid#381] [Ideas]
I've been getting used to Retroarch recently, gotta say, the shaders made me try to love it lol. Anyway, I'm only asking about these to estimate how likely it is to get added or how long it will take to implement. A man can dream, right?

**Comment by mffakhri** [comment]
Almost forgot, maybe choosing cores for each console can be applied and simplified in Lemuroid.

**Reply by ghost** [reply]
Hello,
I found this idea of ​​choosing cores like choosing between desmume and melonDS quite interesting, for example choosing the ParallelN64 core which is fast and accurate.

**Reply by mffakhri** [reply]
> Hello, I found this idea of ​​choosing cores like choosing between desmume and melonDS quite interesting, for example choosing the ParallelN64 core which is fast and accurate.

Yessss, this is exactly why. And based on my experience with Sega Genesis on Lemuroid, it has video filter(?) you can choose Composite to simulate the nostalgic composite look. It seems like some core has video filter(?)
